libraries/AC_CustomControl/AC_CustomControl.cpp

```diff
@@ -0,0 +1,183 @@
+#include <AP_HAL/AP_HAL.h>
+
+#include "AC_CustomControl.h"
+
+#if CUSTOMCONTROL_ENABLED
+
+
+#include "AC_CustomControl_Backend.h"
+// #include "AC_CustomControl_Empty.h"
+#include "AC_CustomControl_PID.h"
+
+// table of user settable parameters
+const AP_Param::GroupInfo AC_CustomControl::var_info[] = {
+    // @Param: CONT_TYPE
+    // @DisplayName: Attitude control type
+    // @Description: Attitude control type to be used
+    // @Values: 0:None, 1:Empty, 2:PID
+    // @RebootRequired: True
+    // @User: Advanced
+    AP_GROUPINFO_FLAGS("_CONT_TYPE", 1, AC_CustomControl, _controller_type, 0, AP_PARAM_FLAG_ENABLE),
```

Comment:
personally I'd drop the "_CONT_" part of this variablae and just make it "CC_TYPE".